aws/terraform/storage.rego

```diff
@@ -1210,6 +1309,45 @@ s3_website_metadata := {
     "Resource Help URL": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html"
 }
 
+
+#
+# PR-AWS-TRF-V4-S3-013
+#
+
+default s3_website_v4 = null
+
+aws_issue["s3_website_v4"] {
+    resource := input.resources[i]
+    lower(resource.type) == "aws_s3_bucket_website_configuration"
+    resource.properties.bucket
+}
+
+s3_website_v4 {
+    lower(input.resources[i].type) == "aws_s3_bucket_website_configuration"
+    not aws_issue["s3_website_v4"]
+}
+
+s3_website_v4 = false {
+    aws_issue["s3_website_v4"]
+}
+
+s3_website_v4_err = "S3 buckets with configurations set to host websites" {
+    aws_issue["s3_website_v4"]
+}
+
+s3_website_v4_metadata := {
+    "Policy Code": "PR-AWS-TRF-V4-S3-013",
```

Comment:
@harshilswan naming convention should be the same.
do not add `v4` to the code
you can use `13v4`

Reply:
or `s3v4`

Reply:
ok